src/test/unit/transponder_ir_unittest.cc

```diff
@@ -60,7 +60,7 @@ TEST(transponderTest, updateTransponderDMABufferArcitimer) {
     transponder_t transponder;
     transponder.dma_buffer_size = TRANSPONDER_DMA_BUFFER_SIZE_ARCITIMER;
     transponder.bitToggleOne = 78;
-    memset(&(transponder.transponderIrDMABuffer.arcitimer), 0, TRANSPONDER_DMA_BUFFER_SIZE_ARCITIMER);
+    memset(&(transponder.transponderIrDMABuffer.arcitimer), 0, sizeof(transponder.transponderIrDMABuffer.arcitimer));
```

Comment:
I think this is unrelated to gcc. Should this go into a separate pull request? Also, is this define now unused and should it be removed?

Reply:
I allready descriped above why this change is needed. 
also of coure the marko is still in use. transponderIrDMABuffer uses this to determine the array size